### DISCUSSION TITLE
dahliaOS Linux 220222

### DISCUSSION THREAD

**Discussion by nmcain** [dahliaOS/releases#10] [Announcements]
# dahliaOS Linux 220222
It's been quite a while since we've released an update to dahliaOS, but the day has come! We've spent the last year hard at work preparing  our latest and greatest release! Here's what's changed!

P.S. If you like what we've done, feel free to help us out on Github or donate to us on Open Collective! 
[https://opencollective.com/dahliaos](https://opencollective.com/dahliaos)
## Changelog

* Desktop Redesign: Pangolin has been completely rebuilt, with new features and functionality
* New Search Functionality: You can now search through apps quickly, speeding up multitasking.
* Improved Launcher: The launcher has been redesigned to separate applications into different categories
* Improved Quick Settings Menu: The Quick Settings menu has been redesigned to prioritize the viewing of relevant information and configuration options. 
* Improved Taskbar: Applications can be pinned and unpinned, and windows can be minimized en-masse using the Show Desktop button.
* New Window Manager: A new window manager for Pangolin, Utopia, improves the handling of windows and fixes memory issues caused by the previous window management solution.
* Improved Applications: Files, Settings, Terminal, and Calculator have received significant updates to functionality and visuals.
* Working web browsing and web applications: After a long struggle with getting web browsing to work, we've added a new web app runtime that will enable you to run web apps smoothly
* Select web apps can now be installed from the new web app manager

* The kernel has been updated to 5.17-rc5
* The system now supports a variety of virtualization solutions, including Linux under QEMU and Fuchsia under FImage
* Included binaries for dap and web_runtime
* The system has been moved over to BTRFS for increased speed and stability
* Improved networking stack: We've moved dahliaOS Linux over to using network-manager for faster scanning and easier configuration.

## Known Issues
* This build may not work on certain AMD hardware. Devices with AMD CPUs and NVIDIA GPUs should work, but devices with AMD GPUs may fail to boot.
* Likewise, limited performance may be experienced on NVIDIA GPUs, due to issues with the nouveau driver.
* This build may also not boot on certain machines with specific EFI configurations (e.g. Apple). The EFI release does not load on these devices, but the legacy image should continue to work.
* Disk I/O on slower live storage devices is significant, we recommend using USB3.0 or better for the EFI image, the legacy image boots from memory, so make sure you have at least 4GB available.
## Screenshots
![image](https://user-images.githubusercontent.com/18690588/155258127-05f7958e-e904-448e-9754-d93779acae97.png)
![image](https://user-images.githubusercontent.com/18690588/155258252-5fd7f548-ab39-4c09-8b1f-99701dc5ca02.png)
![image](https://user-images.githubusercontent.com/18690588/155258306-115c89d2-ee54-4343-9bc2-ec8393c2cb55.png)
![image](https://user-images.githubusercontent.com/18690588/155258689-b746c715-0b18-491b-a9a7-2cab53d7bbfa.png)
![image](https://user-images.githubusercontent.com/18690588/155258805-069e1f86-7d4f-4a6d-badf-b358a8b6addc.png)
![image](https://user-images.githubusercontent.com/18690588/155261338-80ffe33a-399c-43b7-811a-b1f1b377a707.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/dahliaOS/releases/releases/tag/220222-x86_64'>dahliaOS Linux 220222</a>.</em>

**Comment by d4f5409d** [comment]
Will we get Gnome Software? And Flathub support?

**Reply by quintenvandamme** [reply]
We will prob get flatpak to work at some point but no plans for Gnome Software.

**Reply by d4f5409d** [reply]
Thanks for the answer. After 3 months, I probably don't regret you won't support Gnome Software. :)

**Comment by rasit** [comment]
I created bootable USBs with Etcher (on big sur) , but both gives Missing OS on my mid 2012 macbook. What am I missing?

**Reply by nmcain** [reply]
See known issue number 3. I have this issue as well, although, since you have Big Sur, you can use UTM or VMWare Fusion to virtualize the legacy image, which works pretty well.

**Reply by d4f5409d** [reply]
Have you tried Ventoy?

**Reply by marcellotogg** [reply]
I tried on virtual box (Windows 11 host). Same problem, can't boot with efi. Legacy don't work rather.

**Comment by jsj2004** [comment]
Does 4GB requirement is strict or just considered above 2GB, will it work for a 3GB ram computer?

**Comment by Norbit101** [comment]
I'm a graphic designer in progress still learning... I use gimp, inkscape qnd adobe cs6 more often ... Can this OS handle that as a day to day activities?

**Reply by carbonatedcaffeine** [reply]
The desktop isn't ready for that kind of thing yet. Eventually yes it will be.

**Reply by Norbit101** [reply]
Okay then thank you

**Reply by d4f5409d** [reply]
I can't wait to ise this as my daily driver! Finally a Linux distro, with a beautiful Desktop Environment!

**Reply by Norbit101** [reply]
I know right 😀

**Reply by Abujba** [reply]
How to install the system is possible to explain

**Comment by DjagbleyEmmanuel** [comment]
Definitely installing this as my daily driver. btw, about android apps support, Is it happening?

**Reply by gopelkujo** [reply]
are you still using this os as your daily driver? i really interested to this linux since the DE is really beautiful. But because I'm a programmer so i need to know about the software support.

**Comment by vikash52** [comment]
Will I be able to use this OS in this life? More precisely before I retire in 2050?

**Comment by fr4style** [comment]
Is it working on Raspberry PI 4 - 2GB? I flashed the EFI downloaded on the website, but the RPI4 doesn't boot! it's stuck in the boot screen trying to load some USB stuff...

![PXL_20230310_165249437](https://user-images.githubusercontent.com/7868283/224377459-f7416ef6-4057-4767-8393-6e9b739119bc.jpg)

**Comment by gopelkujo** [comment]
Hello it's been 2 year since this release uploaded, i've been waiting for years to see this linux update. I really interested to this beautiful linux distro, so if you don't mind i'd like to ask that is dahliaOS still active in development?

**Reply by carbonatedcaffeine** [reply]
Hi, we aren't actively developing the operating system. Some of the reasons being that we are burned out, working on other projects and or studying.

Thanks.

**Reply by gopelkujo** [reply]
Ow hi! thanks for the information, i know greatness takes time🔥 salute for u guys maintainer!

**Comment by rgiuffre** [comment]
OMG! A Linux distro that hasn't been updated for a long time? Unheard of! Unbelievable. Do you know which operating systems get updated ALL THE TIME? Microsoft's Windows and Apple's MAC-OS. Can you imagine?

**Reply by carbonatedcaffeine** [reply]
I know right!

**Comment by grasenwarr** [comment]
how do i install this on my pc
